src/Object.h

```diff
@@ -45,6 +45,11 @@ namespace sdbus::internal {
     public:
         Object(sdbus::internal::IConnection& connection, std::string objectPath);
 
+        const std::string& getObjectPath() const override
+        {
+            return objectPath_;
+        }
```

Comment:
The order of methods here doesn't matter much already, but I would also move it after all other virtual methods to keep ordering consistency.

Reply:
Done